exchange/bidder.go

```diff
@@ -327,23 +327,39 @@ func getAssetByID(id int64, assets []nativeRequests.Asset) (nativeRequests.Asset
 
 // makeExt transforms information about the HTTP call into the contract class for the PBS response.
 func makeExt(httpInfo *httpCallInfo) *openrtb_ext.ExtHttpCall {
+	if httpInfo == nil {
+		return &openrtb_ext.ExtHttpCall{}
+	}
+
 	if httpInfo.err == nil {
-		return &openrtb_ext.ExtHttpCall{
-			Uri:            httpInfo.request.Uri,
-			RequestBody:    string(httpInfo.request.Body),
-			ResponseBody:   string(httpInfo.response.Body),
-			Status:         httpInfo.response.StatusCode,
-			RequestHeaders: httpInfo.request.Headers,
+		if httpInfo.request != nil && httpInfo.response != nil {
+			return &openrtb_ext.ExtHttpCall{
+				Uri:            httpInfo.request.Uri,
+				RequestBody:    string(httpInfo.request.Body),
+				ResponseBody:   string(httpInfo.response.Body),
+				Status:         httpInfo.response.StatusCode,
+				RequestHeaders: filterHeader(httpInfo.request.Headers),
+			}
 		}
-	} else if httpInfo.request == nil {
-		return &openrtb_ext.ExtHttpCall{}
 	} else {
-		return &openrtb_ext.ExtHttpCall{
-			Uri:            httpInfo.request.Uri,
-			RequestBody:    string(httpInfo.request.Body),
-			RequestHeaders: httpInfo.request.Headers,
+		if httpInfo.request != nil {
+			return &openrtb_ext.ExtHttpCall{
+				Uri:            httpInfo.request.Uri,
+				RequestBody:    string(httpInfo.request.Body),
+				RequestHeaders: filterHeader(httpInfo.request.Headers),
+			}
 		}
 	}
+
+	return &openrtb_ext.ExtHttpCall{}
+}
+
+var authorizationHeader = http.CanonicalHeaderKey("authorization")
```

Comment:
We should make this a constant variable as we don't expect any code to change it

Reply:
In Go, constants must be a hardcoded value. The function call to `CanonicalHeaderKey` prevents me from marking it as a constant.